tools/dev/prometheus_config/prometheus.yml

```diff
@@ -1,11 +1,20 @@
-scrape_configs:
-- job_name: weaviate
-  scrape_interval: 2s
-  static_configs:
-  - targets:
-    - host.docker.internal:2112
-- job_name: node
+global:
   scrape_interval: 5s
-  static_configs:
-  - targets:
-    - host.docker.internal:9100
```

Comment:
Hey @moogacs, the reason we had 9100 was for local node_exporter metrics (`brew install node_exporter && brew services start node_exporter`) as it is useful to see disk/mem/cpu utilization of the host.